### ISSUE TITLE
Precommit wants python 3.6, but i have 3.7

### ISSUE THREAD

**Issue by sofroniewn** [napari/napari#328]
## 🐛 Bug

I get the following error on making commits after installing our precommit hook

```
An unexpected error has occurred: CalledProcessError: Command: ('/anaconda3/bin/python', '-mvirtualenv', '/Users/nicholassofroniew/.cache/pre-commit/repokb02zxfw/py_env-python3.6', '-p', 'python3.6')
Return code: 3
Expected return code: 0
Output:
    The path python3.6 (from --python=python3.6) does not exist

Errors: (none)
```

This is likely because I'm using Python 3.7. I think we should be able to support both. @AhmetCanSolak can you fix this?

**Comment by sofroniewn** [comment]
This can be fixed by changing the setting to `python3` instead of `python3.6` which was required for me to merge my commit. Does anyone have any objections to this solution?

**Comment by jni** [comment]
None here.

It was "fun" seeing all the black commits zooming past btw. =P

**Comment by sofroniewn** [comment]
resolved in #324 by switching to `python3`
